index/scorch/stats.go

```diff
@@ -100,6 +100,8 @@ type Stats struct {
 	TotMemMergeZapBeg   uint64
 	TotMemMergeZapEnd   uint64
 	TotMemMergeSegments uint64
+
+	TotCompactionWrittenBytes uint64
```

Comment:
naming thin -- i think in scorch we don't usually use the word "compaction", even though that's a legit way to think of what the file merger is doing.

How about "TotFileMergeWrittenBytes", and move this stat up a few lines into the file merge related clumping of stats?  (but keep that backwards compat naming alias elsewhere that's "total_compaction_written_bytes"?